spec/commands/lock_spec.rb

```diff
@@ -360,3 +360,24 @@ def read_lockfile(file = "Gemfile.lock")
     end
   end
 end
+
+RSpec.describe "bundle update --bundler" do
+  it "updates the bundler version in the lockfile without re-resolving" do
+    build_repo4 do
+      build_gem "rack", "1.0"
+    end
+
+    install_gemfile! <<-G
+      source "file:#{gem_repo4}"
+      gem "rack"
+    G
+    lockfile lockfile.sub(/(^\s*)#{Bundler::VERSION}($)/, '\11.0.0\2')
+
+    FileUtils.rm_r gem_repo4
+
+    bundle! :update, :bundler => true, :verbose => true
```

Comment:
Should be actually testing `bundle lock --update --bundler` here? Maybe we can add a test similar to 

https://github.com/bundler/bundler/blob/d0103bf094fbde0f38955862872c245e847612d4/spec/commands/lock_spec.rb#L78-L85

but for `bundle lock --update --bundler` instead?